apis/externalsecrets/v1alpha1/secretstore_vault_types.go

```diff
@@ -161,3 +168,20 @@ type VaultJwtAuth struct {
 	// authenticate with Vault using the JWT/OIDC authentication method
 	SecretRef esmeta.SecretKeySelector `json:"secretRef,omitempty"`
 }
+
+// VaultJwtAuth authenticates with Vault using the JWT/OIDC authentication
+// method, with the role name and token stored in a Kubernetes Secret resource.
+type VaultCertAuth struct {
+	// ClientCert is a certificate to authenticate using the Cert Vault
+	// authentication method
+	// +optional
+	ClientCert string `json:"clientCert"`
+
+	// CACert is the Vault TLS Listener CA certificate
+	// +optional
+	CACert string `json:"caCert"`
```

Comment:
I think this is already covered by vault.CABundle here:

https://github.com/external-secrets/external-secrets/blob/7de9dc98c516dd104a28d670de0d10989711c46d/pkg/provider/vault/vault.go#L230-L238

Reply:
Thank you! I wasn't sure that would be kept in memory, but it is.